hivemq4/base-image/entrypoints.d/30_configure_permissions.sh

```diff
@@ -1,20 +0,0 @@
-#!/usr/bin/env bash
-
-# Step down from root privilege, only when we're attempting to run HiveMQ though.
-if [[ "$1" = "/opt/hivemq/bin/run.sh" && "$(id -u)" = '0' && "${HIVEMQ_NO_ROOT_STEP_DOWN}" != "true" ]]; then
```

Comment:
There are some references about this root step down in the readme file. Let's also clean that up.

Reply:
Removed the sections from the readme